src/main/java/frc/robot/RobotContainer.java

```diff
@@ -266,9 +259,16 @@ private void configureButtonBindings() {
         .pov(270)
         .whileTrue(climber.raiseOrLower(ClimberConstants.CLIMBER_BACKWARD_PERCENT));
 
-    driverController.rightBumper().whileTrue(arm.toSetpoint(ArmConstants.STOW));
+    // driver controller
+    driverController.rightBumper().onTrue(arm.toSetpoint(ArmConstants.STOW));
     driverController.leftBumper().onTrue(orchestrator.alignArmAmp());
-    //    driverController.leftBumper().whileTrue(orchestrator.alignArmSpeaker());
+    driverController.leftTrigger().toggleOnTrue(orchestrator.intakeBasic());
+    driverController.rightTrigger().onTrue(orchestrator.indexBasic());
```

Comment:
What's the purpose of having the same command with toggle on left trigger and press on right trigger?

Reply:
intake runs intake and indexer and stops when we have a note. index just pushes the note after the flywheel has spun up

Reply:
this is how akhil wanted it